prometheus/doc/alert-manager.md

```diff
@@ -0,0 +1,53 @@
+OpenPai leverage [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to do
+alert. Alertmanager can send administrator an email to indicate cluster failure or problem with
+pre-configured rules.
+
+# Configuration
+
+To enable alert manager, you should configure it correctly:
+* select a node to deploy alert manager, it can be a master node of kubernetes or any other node. Once you selected, you should mark this node with `alert-manager: "true"` in `cluster-configuration` file, as [example](../../cluster-configuration/cluster-configuration.yaml)
+* configure alert manager by adding `alerting` fields under `prometheus` to services-configuration file. See [example configuration](../../cluster-configuration/services-configuration.yaml).
+
```

Comment:
To facilitate the OpenPAI usage, we had predefined few alerting fields for the system.